src/mpc-hc/AppSettings.cpp

```diff
@@ -2105,6 +2105,8 @@ void CAppSettings::ParseCommandLine(CAtlList<CString>& cmdln)
                 nCLSwitches |= CLSW_ADD;
             } else if (sw == _T("randomize")) {
                 nCLSwitches |= CLSW_RANDOMIZE;
+            } else if (sw == _T("mute")){

```

Comment:
Why add here? Keep some sane order. Probably end will be good to make it roughly follow enum order.